platforms/android/tangram/src/main/cpp/platform_android.cpp

```diff
@@ -231,6 +233,21 @@ void AndroidPlatform::setContinuousRendering(bool _isContinuous) {
     jniEnv->CallVoidMethod(m_tangramInstance, setRenderModeMethodID, _isContinuous ? 1 : 0);
 }
 
+std::string AndroidPlatform::getAssetPath(const Tangram::Url& _path) const {
+
+    if (_path.scheme() == "asset") {
+        JniThreadBinding jniEnv(jvm);
+        auto assetPath = _path.path()[0] == '/' ? _path.path().substr(1) : _path.path();
```

Comment:
What @matteblair said. And I did not look at the spec, but just the code we have in our `Url` handling: https://github.com/tangrams/tangram-es/blob/master/core/src/util/url.cpp#L350-L361